cardano-node/src/Cardano/CLI/Ops.hs

```diff
@@ -214,6 +215,7 @@ data CliError
   | GenesisReadError !FilePath !Genesis.GenesisDataError
   | GenesisSpecError !Text
   | IssueUtxoError !RealPBFTError
+  | NoBlocksFound FilePath
```

Comment:
Not a big deal but all of these constructors with `FilePath` params specify it as strict so:

```suggestion
  | NoBlocksFound !FilePath
```